LoveMachine.Core/ButtplugController.cs

```diff
@@ -107,6 +110,31 @@ protected IEnumerator RunVibratorLoop(int girlIndex, Bone bone)
             }
         }
 
+        protected IEnumerator RunRotatorLoop(int girlIndex, Bone bone)
+        {
+            while (true)
+            {
+                if (IsIdle(girlIndex))
+                {
+                    yield return new WaitForSeconds(.1f);
+                    continue;
+                }
+                string pose = GetPose(girlIndex);
+                if (GetPose(girlIndex) != pose)

```

Comment:
This will always be false since there is no yield between the previous line and this.

Reply:
I see. I just copied and pasted stroker's one 🙈